tests/client/core/i18n/test_utils.js

```diff
@@ -445,9 +445,29 @@ describe('i18n utils', () => {
       expect(i18n.formatNumber(9518231)).toEqual('9,518,231');
     });
 
-    it('localised formatting a number', () => {
+    it('provides a number localized by locale', () => {
+      const numberFormatSpy = sinon.spy(Intl, 'NumberFormat');
       const i18n = utils.makeI18n({}, 'de', FakeJed);
       expect(i18n.formatNumber(9518231)).toEqual('9.518.231');
+      sinon.assert.calledWith(numberFormatSpy, 'de');
+    });
+
+    it('calls Intl.NumberFormat.format', () => {
+      const i18n = utils.makeI18n({}, 'de', FakeJed);
+      const fakeNumberFormat = {
+        format: sinon.stub(),
+      };
+      i18n.formatNumber(9518231, { _NumberFormat: fakeNumberFormat });
+      sinon.assert.calledWith(fakeNumberFormat.format, 9518231);
+    });
+
+    it('falls-back to number.toLocaleString if Intl is undefined', () => {
+      const i18n = utils.makeI18n({}, 'de', FakeJed, false);
```

Comment:
How do you do that with flow? I'm not finding any docs that explain how to handle things like options objects with defaults in function signatures? 

e.g: 

```js
function foo(bar, { baz = 'something' } = {}){
 // blah
}
```